frame/support/src/traits.rs

```diff
@@ -347,22 +347,23 @@ impl<T> Happened<T> for () {
 /// be the default value), or where the account is being removed or reset back to the default value
 /// where previously it did exist (though may have been in a default state). This works well with
 /// system module's `CallOnCreatedAccount` and `CallKillAccount`.
-pub struct StorageMapShim<
-	S,
-	Created,
-	Removed,
-	K,
-	T
->(sp_std::marker::PhantomData<(S, Created, Removed, K, T)>);
+pub struct StorageMapShim<S, Created, Removed, K, T>(
+	sp_std::marker::PhantomData<(S, Created, Removed, K, T)>,
+);
 impl<
-	S: StorageMap<K, T, Query=T>,
-	Created: Happened<K>,
-	Removed: Happened<K>,
-	K: FullCodec,
-	T: FullCodec,
-> StoredMap<K, T> for StorageMapShim<S, Created, Removed, K, T> {
-	fn get(k: &K) -> T { S::get(k) }
-	fn is_explicit(k: &K) -> bool { S::contains_key(k) }
+		S: StorageMap<K, T, Query = T>,
+		Created: Happened<K>,
+		Removed: Happened<K>,
+		K: FullCodec,
+		T: FullCodec,
+	> StoredMap<K, T> for StorageMapShim<S, Created, Removed, K, T>
+{
+	fn get(k: &K) -> T {
+		S::get(k)
+	}
+	fn is_explicit(k: &K) -> bool {
+		S::contains_key(k)
+	}
```

Comment:
previous formatting was ok, I usually prefer when we don't touch formatting if we don't change the code,
for instance this will conflict with reference counting PR for no real reason

Also changing these stuffs add some difficulty in the review, because reviewer has to understand what have changed and why, to see that nothing has changed.
(I don't mind in small PR though)

Reply:
I tried to revert most of these. I agree that it is better not to touch formatting of old code, this was a mistake.